frontend/src/metabase/static-viz/components/WaterfallChart/OldWaterfallChart.jsx

```diff
@@ -61,7 +61,8 @@ const VALUES_MARGIN = 6;
  * @param {import("./types").WaterfallChartProps} props
  * @returns {JSX.Element}
  */
-function WaterfallChart({
+// TODO delete this component and file
```

Comment:
For now I'm renaming it so I can reference the old code without switching branches.